spec/models/user_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject { create(:user) }
+
+  describe 'validations' do
+    it { should validate_presence_of(:full_name) }
+  end
```

Comment:
maybe you can add a validation for the associations in a new describe. WDYT?

Reply:
Thanks, i complete forget it!